web_responsive/static/src/less/main.less

```diff
@@ -4,6 +4,7 @@
 body {
     width: 100%;
     height: 100%;
+    overflow: hidden!important; // Important is to override base template
```

Comment:
What @danieltorres7 mean by "base template", is that there's a `style="overflow: auto" in the `<body>` tag ` *directly in the DOM*.

There's no way to add a more specific selector without `!important`